transaction-service/src/main/kotlin/protobuf/GRPCService.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
```

Comment:
proto files should be placed in separate module, so client which will use your API can add dependency only on that module
for more details about naming messages and organising files search 'Internal API conventions (v2)' on confluence

Reply:
renamed